themes/tiddlywiki/vanilla/base.tid

```diff
@@ -1910,25 +1921,62 @@ a.tc-tiddlylink.tc-plugin-info:hover .tc-plugin-info > .tc-plugin-info-chunk > s
 */
 
 .tc-chooser {
-	border: 1px solid <<colour table-border>>;
+	border-right: 1px solid <<colour table-header-background>>;
+	border-left: 1px solid <<colour table-header-background>>;
 }
 
+
 .tc-chooser-item {
-	border: 8px;
-	padding: 2px 4px;
+	border-bottom: 1px solid <<colour table-header-background>>;
+	border-top: 1px solid <<colour table-header-background>>;
+	padding: 2px 4px 2px 14px;
+}
+
+.tc-drop-down .tc-chooser-item {
+	padding: 2px;
+}
+
+.tc-chosen,
+.tc-chooser-item:hover {
+	background-color: <<colour table-header-background>>;
+	border-color: <<colour table-footer-background>>;
+}
+
+.tc-chosen a.tc-tiddlylink {
+	cursor:default;
 }
 
 .tc-chooser-item a.tc-tiddlylink {
 	display: block;
 	text-decoration: none;
-	color: <<colour tiddler-link-foreground>>;
-	background-color: <<colour tiddler-link-background>>;
+	background-color: transparent;
 }
 
-.tc-chooser-item a.tc-tiddlylink:hover {
+.tc-chooser-item:hover a.tc-tiddlylink:hover {
 	text-decoration: none;
-	color: <<colour tiddler-link-background>>;
-	background-color: <<colour tiddler-link-foreground>>;
+}
+
+.tc-drop-down .tc-chosen a.tc-tiddlylink,
+.tc-drop-down .tc-chooser-item a.tc-tiddlylink:hover {
+	color: <<colour foreground>>;
+}
+
+.tc-chosen > a.tc-tiddlylink:before {
```

Comment:
I think: `+.tc-chosen a.tc-tiddlylink:before {`  should also work, with much less impact on the CSS processor. 
the `a` element is allways the first:child below `tc-chosen`